presto-kafka/src/test/java/com/facebook/presto/kafka/TestManySegments.java

```diff
@@ -59,7 +59,7 @@ public void startKafka()
         topicName = "test_" + UUID.randomUUID().toString().replaceAll("-", "_");
 
         Properties topicProperties = new Properties();
-        topicProperties.setProperty("segment.bytes", "256");
+        topicProperties.setProperty("segment.bytes", "1048576");
```

Comment:
The test was failing due to the message set size being too large (```MessageSetSizeException```).  I am not thrilled with the way these tests are setup either but didn't want to muck with everything.

We can reset the config property to the old value and adjust the tests as another approach; I haven't tested that method yet.